src/app/shared/utils/utils.ts

```diff
@@ -438,3 +439,20 @@ function parseAnswerListItem(item: any) {
   }
   return item;
 }
+
+/**
+ * Compiles markdown to HTML synchronously.
+ * Extends the renderer of "marked" plugin to ensure that links open in new tags.
+ * Code from https://github.com/markedjs/marked/pull/1371#issuecomment-434320596
+ */
+export function parseMarkdown(src: string, options?: marked.MarkedOptions) {
+  const renderer = new marked.Renderer();
+  renderer.link = function (href, title, text) {
+    const link = marked.Renderer.prototype.link.apply(this, arguments);
+    return link.replace("<a", "<a target='_blank'");
```

Comment:
nit(non-blocking) should probably add rel links for noopener and noreferrer, i.e. 
```
<a target='_blank' rel='noopener noreferrer'
```
https://pointjupiter.com/what-noopener-noreferrer-nofollow-explained/